src/pages/partners/payment-agent/_p2p_banner.tsx

```diff
@@ -81,30 +81,29 @@ const LeftChild = styled(Flex)`
 
 const StyledHeader = styled(Header)`
     @media ${device.tabletL} {
-        padding: 0 20px 8px 10px;
         font-size: 26px;
     }
 `
 
 const Subtitle = styled(Header)`
     @media (max-width: 1100px) {
         margin-bottom: 24px;
-        padding: 0 20px 0 10px;
+        padding: 0 20px 0 8px;
     }
 `
-const BannerText = styled.span`
-    color: var(--color-red);
-    margin-bottom: 3rem;
-    font-size: 2rem;
-
+const BannerText = styled(Header)`
     @media (max-width: 1100px) {
         margin-bottom: 16px;
     }
 `
 
 const StyledLinkButton = styled(LinkButton)`
-    width: 146px;
+    width: 30%;
     margin-left: 10px;
+
+    @media ${device.tabletL} {
+        width: 55%;
+    }
```

Comment:
I have spoken to designer to update their design. Please change accordingly as well. The regular LinkButton should be sufficient.

Reply:
For now width looks in two lines if we use width: 107px. Other properties the same as in figma. So I decided to change on 111px when the line of string only one